raspicam/README.md

```diff
@@ -1,29 +1,52 @@
-This folder contains some simple camera classification examples specific to Raspberry
-Pi, using the picamera python module to access the camera.
+# OpenCV examples for inferencing on Coral
 
-If you dont have picamera installed you can install it by:
+This folder contains example code using [picamera](https://github.com/waveform80/picamera) to obtain
+camera images and then perform image classification on the Edge TPU.
 
-```
-pip3 install picamera
-```
+This code works on Raspberry Pi with the Pi Camera and Coral USB Accelerator.
 
-Don't forget to enable your camera using raspi-config under "Interfacing Options":
```

Comment:
This seems like a useful thing to point out - i've seen people be confused about why it doesnt work due to this.

Reply:
I replaced this with a link showing the complete walkthru including this step.